velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -2837,3 +2848,86 @@ TEST_F(DateTimeFunctionsTest, dateParse) {
   VELOX_ASSERT_THROW(
       dateParse("116", "%y+"), "Invalid format: \"116\" is malformed at \"6\"");
 }
+
+TEST_F(DateTimeFunctionsTest, dateFunction) {
```

Comment:
Nit : Please split this function into 3 separate ones to make it more readable.

Reply:
Done.